modules/acquia_cms_article/acquia_cms_article.install

```diff
@@ -82,3 +82,70 @@ function acquia_cms_article_update_8003() {
     }
   }
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Update Article display modes.
+ */
+function acquia_cms_article_update_8004() {
```

Comment:
Can we create a service class for hook_update logic? and call that service class where we need? @rajeshreeputra 

As I can notice that every content_type.install hook_update having same logic with different variable names.

cc: @vishalkhode1